go.mod

```diff
@@ -20,6 +20,200 @@ require (
 	sigs.k8s.io/controller-runtime v0.9.2
 )
 
+require (
```

Comment:
I like this change, but let's also make sure to update the non-openshift Dockerfile to make use of the vendor directory (right now, per operator-sdk standard, it does not)

Reply:
it is not my initiative, it is go version 1.17